data/brands/amenity/pharmacy.json

```diff
@@ -2515,8 +2515,81 @@
         "amenity": "pharmacy",
         "brand": "Super-Pharm",
         "brand:wikidata": "Q13220217",
+        "brand:wikipedia": "en:Super-Pharm",
```

Comment:
```suggestion
```

Thank you for editing NSI, but we've deprecated wikipedia tag since last year, you can see https://github.com/osmlab/name-suggestion-index/issues/6481 for more information.

Reply:
Fixed